cmd/template/app/runner.go

```diff
@@ -147,6 +162,10 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 	}
 	appConfig.NamespaceConfigLabels = namespaceLabels
 
+	if r.flag.PreventDeletion {
+		appConfig.ExtraLabels["giantswarm.io/prevent-deletion"] = "true"
```

Comment:
how about extracting the label to a constant?

Reply:
Right. Let's add it in k8smetadata: https://github.com/giantswarm/k8smetadata/pull/83